ray-operator/apis/ray/v1/rayservice_types.go

```diff
@@ -57,6 +66,9 @@ type RayServiceSpec struct {
 	DeploymentUnhealthySecondThreshold *int32 `json:"deploymentUnhealthySecondThreshold,omitempty"`
 	// ServeService is the Kubernetes service for head node and worker nodes who have healthy http proxy to serve traffics.
 	ServeService *corev1.Service `json:"serveService,omitempty"`
+	// RayServiceUpgradeStrategy represents the strategy used when upgrading the RayService. Currently supports `NewCluster` and `None`
+	// +kubebuilder:default:=NewCluster
+	RayServiceUpgradeStrategy RayServiceUpgradeStrategy `json:"upgradeStrategy,omitempty"`
```

Comment:
UpgradeStrategy RayServiceUpgradeStrategy `json:"upgradeStrategy,omitempty"`

Reply:
Thanks!

Reply:
Should be:
```
UpgradeStrategy RayServiceUpgradeStrategy json:"upgradeStrategy,omitempty"
```

Not:
```
RayServiceUpgradeStrategy RayServiceUpgradeStrategy `json:"rayServiceUpgradeStrategy,omitempty"`
```